composer.json

```diff
@@ -10,6 +10,6 @@
         "php": ">=5.4"
     },
     "require-dev": {
-        "phpunit/phpunit": "^6.1"
+        "phpunit/phpunit": "^6.1 || ^7.0 || ^8.0"
```

Comment:
If this works with PHPUnit 8, let's remove the PHPUnit 6 and 7 options (since the `require-dev` section doesn't affect code that uses this library, it only affects the development and testing of this library itself).